front-issuer/src/contexts/tpf-context.js

```diff
@@ -327,6 +331,29 @@ export const TPFProvider = (props) => {
     }
   }
 
+  const transfer = async ({ contractAddress, quantity, signer, to }) => {
+    try {
+
+      const abi = 'function transfer(address, uint256)';
+      const erc20Contract = new ethers.Contract(contractAddress, [abi], signer);
```

Comment:
Usar o ABI importado no topo do arquivo
```suggestion
      const erc20Contract = new ethers.Contract(contractAddress, TPF_ABI, signer);
```